tests/yatai/client/test_bento_repository_api.py

```diff
@@ -1,150 +1,65 @@
-            {'this_is_a_super_long_key_name_it_will_be_more_than_the_max_allowed': 'v'}
-        )
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key_contains!': 'value'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key': 'value-contains?'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key nop': 'value'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key': '1', 'key3@#!$': 'value'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key': 'cant_end_with_symbol_'})
-
-
-def test_validate_labels_pass():
-    _validate_labels({'long_key_title': 'some_value', 'another_key': "value"})
-    _validate_labels({'long_key-title': 'some_value-inside.this'})
-    _validate_labels({'create_by': 'admin', 'py.version': '3.6.8'})
-
-
-def wait_until_container_ready(container_name, check_message, timeout_seconds=120):
-    docker_client = docker.from_env()
-
-    start_time = time.time()
-    while True:
-        time.sleep(1)
-        container_list = docker_client.containers.list(filters={'name': container_name})
-        logger.info("Container list: " + str(container_list))
-        if not container_list:
-            # Raise timeout, if exceeds timeout limit
-            if time.time() - start_time > timeout_seconds:
-                raise TimeoutError(f'Get container: {container_name} timed out')
-            else:
-                continue
-
-        assert (
-            len(container_list) == 1
-        ), f'should be exact one container with name {container_name}'
-
-        if check_message in container_list[0].logs():
-            break
-
-
-@pytest.fixture(scope='module')
-def yatai_server_container():
-    ensure_docker_available_or_raise()
-
-    yatai_docker_image_tag = f'bentoml/yatai-service:{LAST_PYPI_RELEASE_VERSION}'
-    container_name = f'e2e-test-yatai-service-container-{uuid.uuid4().hex[:6]}'
-    port = '50055'
-    command = [
-        'docker',
-        'run',
-        '--rm',
-        '--name',
-        container_name,
-        '-e',
-        'BENTOML_HOME=/tmp',
-        '-p',
-        f'{port}:{port}',
-        '-p',
-        '3000:3000',
-        yatai_docker_image_tag,
-        '--grpc-port',
-        port,
-    ]
-
-    logger.info(f"Starting docker container {container_name}: {command}")
-    docker_proc = subprocess.Popen(
-        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE
-    )
-    wait_until_container_ready(
-        container_name, b'* Starting BentoML YataiService gRPC Server', 120
+if sys.platform == "darwin" or not psutil.POSIX:
+    pytest.skip(
+        "Requires docker, skipping test for Mac OS on Github Actions",
+        allow_module_level=True,
     )
 
-    yield f'localhost:{port}'
 
-    logger.info(f"Shutting down docker container: {container_name}")
-    os.kill(docker_proc.pid, signal.SIGINT)
+@pytest.fixture(scope='session')
+def local_yatai_container_url():
```

Comment:
can we rename these fixtures and functions as suggested in last review?

local_yatai_service_container => yatai_service_container
local_yatai_container_url => yatai_service_url